frontend/views/containers/chatroom/ViewArea.vue

```diff
@@ -0,0 +1,94 @@
+<template lang='pug'>
+.c-view-wrapper
+  p.c-view-label
+    i18n(
+      :args='{title: title, ...LTags("b")}'
+    ) You are viewing {b_} # {title}{_b}
+  .c-view-actions-wrapper
+    button.button.is-small.is-outlined(
+      @click='see'
+      data-test='channelDescription'
+    )
+      i18n Channel Description
+    button-submit.is-success.is-small(
+      @click='join'
+      data-test='joinChannel'
+    )
+      i18n Join Channel
+</template>
+
+<script>
+import sbp from '~/shared/sbp.js'
+import { mapState, mapGetters } from 'vuex'
+import ButtonSubmit from '@components/ButtonSubmit.vue'
+
+export default ({
+  name: 'ViewArea',
+  components: {
+    ButtonSubmit
+  },
+  computed: {
+    ...mapState(['currentGroupId']),
+    ...mapGetters(['currentChatRoomId'])
+  },
+  props: {
+    title: String
+  },
+  methods: {
+    join: async function () {
+      try {
+        await sbp('gi.actions/group/joinChatRoom', {
+          contractID: this.currentGroupId,
+          data: { chatRoomID: this.currentChatRoomId }
+        })
+      } catch (e) {
+        console.error('ViewArea join() error:', e)
```

Comment:
Same here

Reply:
Fixed using alert.

Reply:
Where is the alert? Is this in a change that hasn't been pushed yet?

Reply:
Not pushed yet.
`alert(e.message)`